source/codegen/metadata/nitclk/nitclk.proto

```diff
@@ -0,0 +1,195 @@
+
```

Comment:
Same as the comment in nidigital, we're not including the proto files generated in hapigen here yet. You can just delete this file out of source/codegen/metadata/nitclk